sdk/iOS/ZumoE2ETestApp/ZumoE2ETestApp/ZumoAppDelegate.m

```diff
@@ -12,6 +12,10 @@ @implementation ZumoAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     if([application respondsToSelector:@selector(registerForRemoteNotifications)]) {
+        
+        UIUserNotificationSettings *notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
+        [application registerUserNotificationSettings:notificationSettings];
+        
         [application registerForRemoteNotifications];
```

Comment:
hmm, ok.  I see Apple shows the same pattern, if we make a new revision, maybe add a log in the userFail just for completeness, but ok as is then.